apps/agreement/contracts/Agreement.sol

```diff
@@ -486,6 +485,7 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
     */
     function submitEvidence(uint256 _disputeId, bytes _evidence, bool _finished) external {
         (, Action storage action, , Challenge storage challenge) = _getDisputedAction(_disputeId);
+        // Note: should we also check that the evidence period hasn't already been closed?
```

Comment:
Last note: as discussed offline, we have a simplification in mind for submitting evidence :).

Reply:
Good catch! Addressed in ac9ada497f10e6edd9a72a17c4b9839f498c34f0